demos/v1/emojivoto/spec.yaml

```diff
@@ -0,0 +1,15 @@
+applicationCreator: linkerd
+type: DEMO
+logoUrl: https://avatars2.githubusercontent.com/u/10824329?s=400&v=4
+longDescription: |
+  The canonical testing application for linkerd.
+name: emojivoto
+shortDescription: |
+  The canonical testing application for linkerd.
```

Comment:
could you add a little more detail here? doesn't have to be much, just a little more color.